config/initializers/assets.rb

```diff
@@ -1,12 +1,10 @@
 # Be sure to restart your server when you modify this file.
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.0'
+Rails.application.config.assets.version = "1.0"
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-# Add Yarn node_modules folder to the asset load path.
-Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
Have not noticed issues with this, but interested if there's a reason this was here/removed that isn't driven by the removal of sprockets

Reply:
Pretty sure this would only impact sprockets, which we don't use so it should be fine